<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_Teachers</class>
 <widget class="QDialog" name="edit_Teachers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_16">
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="Assign">
     <property name="text">
      <string>Assign:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_3">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_3</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_3">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_3</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_3">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_3</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_1">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_1">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_1">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_2">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_2">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_2">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_4">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_4</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_4">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_4</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_4">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_4</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" rowspan="6">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="g1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_6">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_6</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_6">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_6</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_6">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_6</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_5">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_5</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_5">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_5</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_5">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_5</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="6">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="g7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g8">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g9">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g10">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g11">
           <property name="text">
            <string>11</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="g12">
           <property name="text">
            <string>12</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_7">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_7</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_7">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_7</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_7">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_7</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_12">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_12</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_12">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_12</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_12">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_12</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_9">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_9</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_9">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_9</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_9">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_9</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_8">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_8</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_8">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_8</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_8">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_8</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_11">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_11</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_11">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_11</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_11">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_11</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <widget class="QRadioButton" name="course_10">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_10</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="lab_10">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_10</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="tuto_10">
           <property name="text">
            <string/>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_10</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Full Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>E-mail :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Password :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Phone : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Address :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="Line_name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_email">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_password">
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_phone">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_address">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="CloseTeacherbtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditTeacherbtn">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <widget class="QListView" name="List_courses"/>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>191</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4"/>
   </item>
   <item row="4" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="Date">
       <property name="text">
        <string>Date Of Birth :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="L_Day"/>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QLabel" name="Year">
         <property name="text">
          <string>Year :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="L_Year"/>
       </item>
       <item row="1" column="1" rowspan="2">
        <widget class="QLineEdit" name="L_Month"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Day">
         <property name="text">
          <string>Day :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Month">
         <property name="text">
          <string>Month :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_5"/>
  <buttongroup name="buttonGroup_6"/>
  <buttongroup name="buttonGroup_7"/>
  <buttongroup name="buttonGroup_8"/>
  <buttongroup name="buttonGroup_9"/>
  <buttongroup name="buttonGroup_10"/>
  <buttongroup name="buttonGroup_11"/>
  <buttongroup name="buttonGroup_12"/>
 </buttongroups>
</ui>
